openwhisk/openwhisk.yaml

```diff
@@ -0,0 +1,100 @@
+# Task to define Runtime Parameters
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+    name: openwhisk
+spec:
+    inputs:
+        resources:
+            - name: runtime-git
+              type: git
+        params:
+            - name: TARGET_IMAGE_NAME
+              description: name of the image to be tagged and pushed
+            - name: TARGET_IMAGE_TAG
+              description: tag the image before pushing
+              default: "latest"
+            - name: DOCKERFILE
+              description: The path to the dockerfile to build
+            - name: OW_RUNTIME_DEBUG
+              description: flag to indicate debug mode should be on/off
+              default: "false"
+            - name: OW_RUNTIME_PLATFORM
+              description: flag to indicate the platform, one of ["openwhisk", "knative", ... ]
+              default: "knative"
+            - name: OW_ACTION_NAME
```

Comment:
I think it would be useful to have here or in the README a pointer to the openwhisk documentation that explains how to setup these parameters, action name, code, main, binary

Reply:
added a link to OpenWhisk publication which has the explanation to all parameters.